python/src/plato/cli.py

```
@@ -1064,6 +564,43 @@ async def handle_create_snapshot(sandbox: Sandbox):
+        console.print("   pip install --upgrade --force-reinstall plato-sdk")
+        console.print("\n[dim]If the issue persists, please report it at:[/dim]")
+        console.print("[dim]https://github.com/plato-app/plato-client/issues[/dim]")
+        raise typer.Exit(1)
+
+    # Get any additional arguments passed after 'hub'
+    args = ctx.args if hasattr(ctx, 'args') else []
+
+    try:
+        # Launch the Go CLI, passing through all arguments
+        # Use execvp to replace the current process so the TUI works properly
+        os.execvp(plato_bin, [plato_bin] + args)
```

Ruff (0.14.1)

Starting a process without a shell

(S606)